### PR TITLE
updated changelog for 2.4.3

### PR DESCRIPTION
**Pull request by wjblanke**
### Purpose:

Information about release

### Current Behavior:

changelog contains nothing about 2.4.3

### New Behavior:

changelog contains something about 2.4.3

### Testing Notes:

N/A

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,72 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
...
+
+* Removed `lerna` from build_scripts
+* (pylint) `==` -> `is`
+* Use tx_config in cancel_pending_trades
+* Fixed an issue where `npx --no lerna clean -f` didn't work
+* Fix for freebsd (thanks @alghmma)
+* Fix backwards compatibility for `add_private_key`
+* Fix timelord log spam.
+* Fix duplicate connection log spam.
+* ad.fee_estimation_fixes
```
Comment (arvidn) [inline CHANGELOG.md:65]: this looks like a branch name

Comment (Starttoaster) [reply CHANGELOG.md:65]: That was the name of the PR https://github.com/Chia-Network/chia-blockchain/pull/18262
The line could probably be the first line of the PR's description:
```suggestion
Fix conversion error in fee calculation
```

**Review by Starttoaster — CHANGES_REQUESTED** [review]
pre-commit is arguing about the bullet points here, where `*` was used, but it wants to standardize on `-`

File: CHANGELOG.md (hunk 1 above)
Comment (Starttoaster) [reply CHANGELOG.md]: Suggestion is outdated so I'll resolve this and make a new one. Thanks for pointing this out Arvid!

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,72 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
...
+
+- Removed `lerna` from build_scripts
+- (pylint) `==` -> `is`
+- Use tx_config in cancel_pending_trades
+- Fixed an issue where `npx --no lerna clean -f` didn't work
+- Fix for freebsd (thanks @alghmma)
+- Fix backwards compatibility for `add_private_key`
+- Fix timelord log spam.
+- Fix duplicate connection log spam.
+- ad.fee_estimation_fixes
```
Comment (Starttoaster) [inline CHANGELOG.md:65]:
```suggestion
- Fix conversion error in fee calculation
```
As pointed out by @arvidn this PR was titled in a way that doesn't really describe the change all that well. This just puts a summary of the PR's description in its place.

**Review by Starttoaster — APPROVED** [review]

**Review by arvidn — APPROVED** [review]

**Pull request merged**